trident/utilities.py

```diff
@@ -88,7 +86,7 @@ def download_file(url, progress_bar=True, local_directory=None,
 
     # Set defaults
     if local_filename is None:
-        local_filename = url.split(os.sep)[-1]
+        local_filename = url.split('/')[-1]
```

Comment:
This might be more robust.
```suggestion
        local_filename = os.path.basename(url)
```